Compile failure under Windows

This is a regression following the _helpers.c cleanup.

```
_helperlib.c
c:\t\numba\numba\_math_c99.h(49) : warning C4005: 'atan2f' : macro redefinition
        c:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\INCLUDE\math.h(296) : see previous definition of 'atan2f'
numba/_helperlib.c(185) : warning C4244: 'function' : conversion from 'double' to 'float', possible loss of data
numba/_helperlib.c(185) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
c:\t\numba\numba\mathnames.inc(28) : error C2065: 'Numba_m_asinh' : undeclared identifier
c:\t\numba\numba\mathnames.inc(29) : error C2065: 'Numba_m_asinhf' : undeclared identifier
c:\t\numba\numba\mathnames.inc(31) : error C2065: 'Numba_m_acosh' : undeclared identifier
c:\t\numba\numba\mathnames.inc(32) : error C2065: 'Numba_m_acoshf' : undeclared identifier
c:\t\numba\numba\mathnames.inc(34) : error C2065: 'Numba_m_atanh' : undeclared identifier
c:\t\numba\numba\mathnames.inc(35) : error C2065: 'Numba_m_atanhf' : undeclared identifier
c:\t\numba\numba\mathnames.inc(40) : error C2065: 'Numba_m_expm1' : undeclared identifier
c:\t\numba\numba\mathnames.inc(41) : error C2065: 'Numba_m_expm1f' : undeclared identifier
c:\t\numba\numba\mathnames.inc(61) : error C2065: 'Numba_m_log1p' : undeclared identifier
c:\t\numba\numba\mathnames.inc(62) : error C2065: 'Numba_m_log1pf' : undeclared identifier
c:\t\numba\numba\mathnames.inc(64) : error C2065: 'Numba_m_round' : undeclared identifier
c:\t\numba\numba\mathnames.inc(65) : error C2065: 'Numba_m_roundf' : undeclared identifier
c:\t\numba\numba\mathnames.inc(67) : error C2065: 'Numba_m_trunc' : undeclared identifier
c:\t\numba\numba\mathnames.inc(68) : error C2065: 'Numba_m_truncf' : undeclared identifier
c:\t\numba\numba\mathnames.inc(77) : error C2065: 'Numba_m_atan2f' : undeclared identifier
error: Command "c:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\BIN\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG /Zi -I
c:\t\ATHENA~1\build\athena\vc90_xp\pylib\__envs__\numba-dev\numpy\core\include -Ic:\t\ana\envs\tmpenv\Lib\site-packages\nump
y\core\include -Ic:\t\ATHENA~1\ext\vc90_xp\include -Ic:\t\ATHENA~1\ext\vc90_xp\include\python2.6 /Tcnumba/_helperlib.c /Fobu
ild\temp.win32-2.6\Release\numba/_helperlib.obj" failed with exit status 2
```

--- a/numba/_math_c99.h
+++ b/numba/_math_c99.h
@@ -26,28 +26,26 @@ float m_atan2f(float y, float x);
 
 #ifdef _MSC_VER
 
-/* define asinh acosh atanh for windows */
+/* Define missing (C99) math functions for Windows */
 
-#define asinh m_asinh
-#define asinhf m_asinhf
-#define acosh m_acosh
-#define acoshf m_acoshf
-#define atanh m_atanh
-#define atanhf m_atanhf
+#define asinh(x) m_asinh(x)
+#define asinhf(x) m_asinhf(x)
+#define acosh(x) m_acosh(x)
+#define acoshf(x) m_acoshf(x)
+#define atanh(x) m_atanh(x)
+#define atanhf(x) m_atanhf(x)
 
+#define expm1(x) m_expm1(x)
+#define expm1f(x) m_expm1f(x)
+#define log1p(x) m_log1p(x)
+#define log1pf(x) m_log1pf(x)
 
-#define expm1 m_expm1
-#define expm1f m_expm1f
-#define log1p m_log1p
-#define log1pf m_log1pf
+#define round(x) m_round(x)
+#define roundf(x) m_roundf(x)
+#define trunc(x) m_trunc(x)
+#define truncf(x) m_truncf(x)
 
-#define round m_round
-#define roundf m_roundf
-#define trunc m_trunc
-#define truncf m_truncf
-
-#define atan2f m_atan2f
-/* provide floating point equivalence */
+#define atan2f(x, y) m_atan2f(x, y)
 
 #endif /* _MSC_VER */
 

